src/runtime/storage/synthetic-storage.ts

```diff
@@ -146,14 +146,13 @@ class SyntheticCollection extends StorageProviderBase implements CollectionStora
     super(type, undefined, id, key);
     this.targetStore = targetStore;
     this.storageFactory = storageFactory;
-    let resolveInitialized;
-    this.initialized = new Promise(resolve => resolveInitialized = resolve);
-    const process = async data => {
-      await this.process(data, false);
-      resolveInitialized();
-      targetStore.on('change', details => this.process(details.data, true), this);
-    };
-    targetStore.get().then(data => process(data));
+
+    const me = this;
```

Comment:
this is unaffected by lambdas, so you can just use `this` instead of `me` below.

Reply:
it's not the case with `function(...) {...}` but this is safe with lambdas. That's AFAIK the only difference in behaviour.

Reply:
Thanks, done!